src/lectures/part3fp/TuplesAndMaps.scala

```diff
@@ -130,7 +130,7 @@ object TuplesAndMaps extends App {
       }
     }
 
-    bfs(b, Set(), network(a) + a)
+    bfs(b, Set(), Set(a))
```

Comment:
How is this better?

Reply:
It seems simpler to just start with `a` instead of a plus all its networks. Its networks will be taken care by the existing code. Getting `a`'s network and including them as the initial parameters is unnecessary. Also it runs faster in case of `bfs(a,a)`.

Reply:
Feel free to close the PR if this does not make sense to you. I think it's simpler in general and faster in a corner case. But I also understand you may think the existing code is clearer which works fine.